### DISCUSSION TITLE
Unable to Open Workflow Instance View Screen in Customized Elsa UI

### DISCUSSION THREAD

**Discussion by SatishKolte** [elsa-workflows/elsa-core#5379] [Q&A]
hello Team,

I am encountering difficulties in opening the Workflow Instance View Screen within a customized Elsa UI. Despite attempts to customize the Elsa UI screen, other screens open correctly. However, when trying to open the Workflow Instance View Screen using the provided code snippet, the desired UI does not appear.

Code Sample:

@page "/"
@inject Microsoft.Extensions.Configuration.IConfiguration Configuration

@{
    Layout = "~/Views/_Layout.cshtml";
    var serverUrl = Configuration["Elsa:Server:BaseUrl"];
}

<elsa-studio-root server-url="@serverUrl" monaco-lib-path="_content/Elsa.Designer.Components.Web/monaco-editor/min">
    <elsa-studio-workflow-instances-view workflow-instance-id="@ViewBag.WorkflowInstanceId" 
</elsa-studio-root

Details:

The server URL is correctly set to https://localhost:5001.
The WorkflowInstanceId is being passed to the elsa-studio-workflow-instances-view tag.
Request:

i want to this type of screen like in following Screen.

![image](https://github.com/elsa-workflows/elsa-core/assets/96614575/9a3c39b2-163c-4062-83a2-83a0e9ed01ed)

![image](https://github.com/elsa-workflows/elsa-core/assets/96614575/882e1e23-6c62-4f72-9b88-c27ce0382b92)

Could you please assist in identifying the appropriate tag to use and how to utilize it to open the desired UI for the Workflow Instance View Screen? A code sample demonstrating the correct implementation would be greatly appreciated.

Thanks, and Regards
Satish

**Comment by glime-ai** [comment]
Hello Satish,

Based on the information provided, it seems that you are trying to integrate the Elsa Workflow Instance View into a custom UI and facing issues with the display of the UI. The code snippet you've provided appears to be correct for embedding the Elsa Workflow Instance View component into a Blazor application.

However, the images you are referring to for comparison are not accessible as the links provided seem to be incorrect or broken. This makes it difficult to understand exactly what UI you are expecting to see.

To troubleshoot the issue, you can check the following:

1. Ensure that the `Elsa:Server:BaseUrl` configuration is set correctly in your `appsettings.json` or wherever you are configuring your server URL.
2. Verify that the `ViewBag.WorkflowInstanceId` is being set correctly and that it corresponds to an existing workflow instance ID in your database.
3. Check the browser's developer console for any JavaScript errors that might indicate problems with loading the Elsa scripts or making API calls.
4. Ensure that all necessary static files for Elsa are being served correctly, and that the path specified in `monaco-lib-path` is correct and accessible.
5. Make sure that the version of the Elsa packages you are using is compatible with your application.

If you continue to face issues, it would be helpful to have access to the actual error messages or console output you are receiving when trying to load the Workflow Instance View. Additionally, providing the version of Elsa you are using and any relevant parts of your `Startup.cs` or `Program.cs` (for .NET 5 and above) could provide more context for further assistance.

For more detailed guidance and support, you may want to check the official Elsa documentation or reach out to the Elsa community on GitHub or other forums where Elsa is discussed.

I hope this helps, and if you have any further details or error messages, please provide them for more specific assistance.

Best regards,
Customer Support Agent

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
